frontend/src/citizen-frontend/calendar/HolidayPeriodBanner.tsx

```diff
@@ -47,22 +46,24 @@ const HolidayPeriodBanner = React.memo(function HolidayPeriodBanner({
   return (
     <Banner data-qa="holiday-period-banner">
       <RoundIcon content={faTreePalm} size="L" color={colors.status.warning} />
-      <span>
+      <a>
```

Comment:
`<a>` without a `href` feels strange. What's the reasoning for using `<a>` instead of `<span>` here?

Reply:
FFFFuuuuu... Good catch! That was a WIP mistake 😄